internal/output/progress_tty.go

```diff
@@ -206,7 +206,13 @@ func (p *progressTTY) writeLine(bar *ProgressBar) {
 
 	// Unicode box drawing gives us eight possible bar widths, so we need to
 	// calculate both the bar width and then the final character, if any.
-	segments := int(math.Round((float64(8*barWidth) * value) / bar.Max))
+	var segments int
+	if bar.Max == 0 {
+		// If we didn't have any progress, we use the max width
+		segments = 8 * barWidth
+	} else {
+		segments = int(math.Round((float64(8*barWidth) * value) / bar.Max))
+	}
```

Comment:
This conflicts with #289. (Flagging as much for @chrispine as you!)

Reply:
Yes, but I committed first. 😛 

Adam and I chatted and thought that if there are zero items, we should leave it blank instead of full width. (Well, Adam thought so. I'm happy either way.)

Reply:
No way! I committed first!

But doesn't matter: happy you found a fix that you like, I wasn't sure what the desired behaviour is.